README.md

```diff
@@ -193,17 +193,3 @@ To rebuild the types the following actions are required (for the why see **IMPOR
 1. (Quit watch mode: `ctrl c`).
 2. Run: `yarn build-types`
 3. Start babel again: `yarn babel:watch`
-
-### Use package in linked (watch-)mode
```

Comment:
This section is still partially valid. E.g. you still have the link the package before you can use the symlink. And the run command just changes it's not gone ;)

Reply:
true, will add back